CHANGELOG.rst

```diff
@@ -1,3 +1,14 @@
+DEV
+----
+
+Improvements:
+
+* return created subscription from Yola.create_subscription (5f7aba284e4678fa10bcc6482c8f04168acc831c)
```

Comment:
I think linking to commits / PRs can compromise the readability of a changelog, without significant benefit.

It should be easy enough to find where something was implemented, if one of:
* releases are small and frequent
* the changelog entry is added together with the feature (in a DEV section, for example)
* commits are well described